src/mozanalysis/experiment.py

```diff
@@ -925,9 +926,7 @@ def _build_metrics_query_bits(
                 """    LEFT JOIN (
         {query}
         ) ds_{i} USING (client_id, branch, analysis_window_start, analysis_window_end)
-                """.format(
-                    query=query_for_metrics, i=i
-                )
+                """.format(query=query_for_metrics, i=i)
```

Comment:
No f-strings in this file?

Reply:
I'm still using the old ruff rules because I'm still waiting on merging the ruff branch.  Waiting for a final approval from one of the DS folks, I'll ask about it today

Reply:
Whoops :facepalm: that makes sense! Glad it's not another strange case of some files skipping some lint rules.